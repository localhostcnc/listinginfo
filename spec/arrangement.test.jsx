/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import Arrangement from '../dist/Components/upperInfo/arrangement';

describe('Components', () => {
  test('Rendering the Arrangment component', () => {
    const wrapper = shallow(
      <Arrangement />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
