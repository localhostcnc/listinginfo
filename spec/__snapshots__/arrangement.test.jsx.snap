// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Rendering the Arrangment component 1`] = `ShallowWrapper {}`;
